CONTRIBUTING.md

```diff
@@ -15,20 +13,18 @@ GitHub.
 ## Contact information
 If you have questions about specific repositories, contact the repository maintainers.
 
-For issues about BioJulia in general, contact the current BioJulia admins:
-- [Kevin Bonham](https://github.com/kescobo)
-- [Jakob Nybo Nissen](https://github.com/jakobnissen)
+For issues about BioJulia in general, please feel free to contact the current [BioJulia admins](https://github.com/orgs/BioJulia/people?query=role%3Aowner).
 
-For quick questions, you can also find us on [The Julia Language Slack](https://julialang.org/slack/) in the `#biology` channel.
+For quick questions, you can also find us on [The Julia Language Slack](https://julialang.slack.com/channels/biology) in the `#biology` channel.
 
 For discussions, the best place is either the [Biology domain on the Julia Language Discourse](https://discourse.julialang.org/c/domain/bio), or our [GitHub discussions page](https://github.com/orgs/BioJulia/discussions).
 
 ## Making an issue
 Issues are meant for bug reports, feature requests, and discussions about potential changes to a particular repository.
 For questions about usage, [ask for help by contacting us instead](#contact-information).
 
-Before filing an issue, make sure to read the documentation of the package in question, and search the issue tracker if the issue you want to raise already exists.
-If the issue exists, but it's closed, you can make a new issue and link to the old, closed issue.
```

Comment:
I think this comma is correct :thinking: https://prowritingaid.com/grammar/1008103/Should-I-use-a-comma-between-two-adjectives

Reply:
That's an excellent article. Could they possibly be cumulative adjectives? The equally descriptive "old and closed", while equally true, doesn't sound correct to me. Anyhow, I stumbled at this comma. Maybe remove "old"?

Reply:
Old **or** closed?